compiler/cpp/src/lexer/Token.h

```diff
@@ -43,6 +43,10 @@ enum TokenType {
     TOK_OR, // |
     TOK_EQUALS, // ==
     TOK_NOT_EQUALS, // !=
+    TOK_GREATER_THEN, // >
```

Comment:
`than` not `then`. Also inconsistent: here you use `TOK_GREATER_THAN` but below without `THAN`: `TOK_GREATER_EQUALS`

Reply:
Ups. Thanks for making me aware ...
I'd say before we have long token type names such as `TOK_GREATER_EQUALS_THAN`, I will rather rename them like so:
- `TOK_GREATER`
- `TOK_LESS`
- `TOK_GREATER_EQUAL`
- `TOK_LESS_EQUAL`